### DISCUSSION TITLE
Where to view latest docs?

### DISCUSSION THREAD

**Discussion by chtenb** [jdan/98.css#210] [Q&A] [answered]
I notice that the documentation in the website is not fully up to date, e.g. does not mention progress bars. Where can I view the latest docs?

**Comment by chanagaray** [comment, marked as answer]
Thanks so much for this question! We have to manually update the docs and I had never even thought of it. Can you confirm that you can see the updated docs now in https://jdan.github.io/98.css/ ?

**Reply by chtenb** [reply]
Yes, thank you!
